index.html

```diff
@@ -43,7 +43,154 @@ <h1>I'm Shahira <br> Glad to see <br>you!</h1>
+                  <li>javaScript</li>
+                </ul>
+               <button class="btn">See Project</button>
+              </div>
+            </section>
+            </div>
+
+            <section class="about-myself" id="about">
+                <div class="desk-about">
+                  <h1>About <br> Myself</h1>
+                  <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                    contact me.</p>
+                  <P class="connect">LET'S CONNECT</P>
+                  <ul class="contact-icon">
+                    <li><a href="#"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+                    <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+                    <li><a href="#"><i class="fa fa-angellist" aria-hidden="true"></i></a></li>
+                    <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+                    <li><a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
+                  </ul>
+                  <button class="resume">Get My Resume</button>
+                </div>
+                <div class="expertise">
+                  <ul class="language">
+                    <li>
+                      <p>Languages</p>
+                    </li>
+                    <li><i class="fa fa-chevron-down" aria-hidden="true"></i></li>
+                  </ul>
+                  <div class="desk-programming">
+                    <ul class="outer-item">
+                      <li class="programming"><img src="image/javaScript.png" alt="JavaScript"></li>
+                      <li>
+                        <p>JavaScript</p>
+                      </li>
+                    </ul>
+                    <ul class="outer-item">
+                      <li class="programming"><img src="image/html.png" alt="HTML"></li>
+                      <li>
+                        <p>HTML</p>
+                      </li>
+                    </ul>
+                    <ul class="outer-item">
+                      <li class="programming"><img src="image/css.png" alt="CSS"></li>
+                      <li>
+                        <p>CSS</p>
+                      </li>
+                    </ul>
+                  </div>
+                  <ul class="framework">
+                    <li class="flex-items">
+                      <p>Frameworks</p>
+                    </li>
+                    <li><i class="fa fa-chevron-right" aria-hidden="true"></i></li>
+                  </ul>
+                  <ul class="skills">
+                    <li class="flex-items">
+                      <p>Skills</p>
+                    </li>
+                    <li><i class="fa fa-chevron-right" aria-hidden="true"></i></li>
+                  </ul>
+                </div>
```

Comment:
- [ ] For this part of the portfolio, I suggest you closely follow the figma design.

Your design:
![image](https://user-images.githubusercontent.com/52143815/170836342-341c031c-2ff1-4aab-b4fa-99d19bf3cc84.png)

Figma design:
![image](https://user-images.githubusercontent.com/52143815/170836363-40c6ac93-165c-43ed-ac03-72cde620eccf.png)